packages/react/src/components/TimeSeriesCard/TimeSeriesCard.jsx

```diff
@@ -532,6 +534,7 @@ const TimeSeriesCard = ({
               options={options}
               width="100%"
               height="100%"
+              key={`thresholds-key${thresholds?.length ? JSON.stringify(thresholds) : ''}`} // have to regen the component if thresholds change
```

Comment:
I think you could use the `objectAgnosticThresholds` var here instead of re-stringifying the thresholds.

Reply:
good point! Fixed